kubernetes-services/work-scheduler/env-defaults

```diff
@@ -0,0 +1,61 @@
+# Port the health check server listens on
+PORT=5000
+
+# Fake credentials for localstack
+AWS_ACCESS_KEY_ID=LOCALSTACK
+AWS_SECRET_ACCESS_KEY=LOCALSTACK
+
+AWS_DEFAULT_REGION=us-west-2
+
+NODE_ENV=production
+
+# For testing: Whether to use Localstack instead of AWS S3.  Options are "true" or "false".
+USE_LOCALSTACK=true
+
+# This should always be true for this service
+USE_SERVICE_QUEUES=true
+
+# Hostname for harmony to use to connect to Localstack
+LOCALSTACK_HOST=localstack
+
+# Hostname for K8S pods to connect to Localstack
+LOCALSTACK_K8S_HOST=localstack
+
+# When set to true log messages are logged as a text string instead of the default
+# JSON format. Useful when running harmony locally and viewing logs via a terminal.
+TEXT_LOGGER=true
+
+# When set to true log all database queries performed by knex
+DEBUG_KNEX=false
+
+# Log level (error, warn, info, debug)
+LOG_LEVEL=debug
+
+# Long polling wait time (in seconds) for queues
+QUEUE_LONG_POLLING_WAIT_TIME_SEC=20
+
+# page size to use with CMR calls - only used for logging in this service
+CMR_MAX_PAGE_SIZE=2000
+
+# SQS queue used to request scheduling of work items for a service
+WORK_ITEM_SCHEDULER_QUEUE_URL=http://localstack:4566/queue/work-item-scheduler-queue.fifo
+SCHEDULER_QUEUE_BATCH_SIZE=10
+# Used to allow slightly more work items to be queued than the number of available workers
+# to avoid queue starvation
```

Comment:
"1.1 for example means to queue 10% more work items than there are running pods for the service."  Assuming that's what that means.

Reply:
Added your sentence to the comment for clarification.